candy-machine/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metaplex-foundation/mpl-candy-machine",
-  "version": "0.0.1",
+  "version": "3.0.1",
```

Comment:
Same here, I cannot see that tag anywhere [here](https://github.com/metaplex-foundation/metaplex-program-library/tags). And as said it would disappear once the PR is squashed.

We want devs to be able to easily find the code corresponding to each npm version (that's why I added those `npm pre version and post publish` scripts.